tests/features/class-wc-tests-shipping-label-banner-display-rules.php

```diff
@@ -0,0 +1,170 @@
+<?php
+/**
+ * Shipping Label Banner Display Rules tests.
+ *
+ * @package WooCommerce\Tests\Shipping-label-banner-display-rules
+ */
+
+use \Automattic\WooCommerce\Admin\Features\ShippingLabelBannerDisplayRules;
+
+/**
+ * Class WC_Tests_Shipping_Label_Banner_Display_Rules
+ */
+class WC_Tests_Shipping_Label_Banner_Display_Rules extends WC_Unit_Test_Case {
+
+	/**
+	 * List of active plugins.
+	 *
+	 * @var array
+	 */
+	private $active_plugins = array();
+
+
+	/**
+	 * Shipping label banner display rules manager.
+	 *
+	 * @var object
+	 */
+	private $shipping_label_banner_display_rules;
+
+	/**
+	 * Setup for tests.
+	 */
+	public function setUp() {
+		parent::setup();
+
+		global $active_plugins;
+		$active_plugins['jetpack'] = 'jetpack/jetpack.php';
+		$active_plugins['wcs']     = 'woocommerce-services/woocommerce-services.php';
+		update_option( 'active_plugins', $active_plugins );
```

Comment:
Does this `update_option` actually perform a DB update during test run?

If we are manipulating the database options in tests, then all subsequent tests run will have a different state for "active plugins". In that case, we may need to restore the original option settings per test case.